app/views/application_letters/check.html.erb

```diff
@@ -18,47 +20,81 @@
 <% end %>
 
 <h3><%= t('.my_application') %></h3>
-<div id='application_details'>
-  <strong><%= ApplicationLetter.human_attribute_name(:grade) %>:</strong>
-  <%= @application_letter.grade %><br/>
-  <strong><%= ApplicationLetter.human_attribute_name(:motivation) %>:</strong><br/>
-  <%= @application_letter.motivation %><br/>
-  <strong><%= ApplicationLetter.human_attribute_name(:coding_skills) %>:</strong><br/>
-  <%= @application_letter.coding_skills %><br/>
-  <strong><%= ApplicationLetter.human_attribute_name(:emergency_number) %>:</strong>
-  <%= @application_letter.emergency_number %><br/>
-  <strong><%= ApplicationLetter.human_attribute_name(:organisation) %>:</strong>
-  <%= @application_letter.organisation %><br/>
-
-  <strong><%=t('.eating_habits') %>:</strong>
-  <%= @application_letter.eating_habits.join(', ') %><br/>
-
-  <strong><%= ApplicationLetter.human_attribute_name(:allergies) %>:</strong><br/>
-  <%= @application_letter.allergies %><br/>
-  <strong><%= ApplicationLetter.human_attribute_name(:annotation) %>:</strong><br/>
-  <%= @application_letter.annotation %><br/>
+<table id='application_details' class="table application-details-table">
+  <col width="250">
+  <tr>
+    <th><%= ApplicationLetter.human_attribute_name(:motivation) %>:</th>
+    <td id="application-motivation" data-orig-text="<%= @application_letter.motivation %>">
+      <%= truncate @application_letter.motivation, length: 300 do link_to I18n.t('events.list.read_on'), 'javascript:jQuery("#application-motivation").text(jQuery("#application-motivation").attr("data-orig-text"))' end %>
```

Comment:
Well, this line is ugly. Since you already have `do` and `end`, maybe you could refactor this into several lines?